bats/examples.bats

```diff
@@ -23,18 +32,18 @@ teardown_file() {
       }
     }'
   )
-
   exec_graphql 'cala-outbox-import-job-create' "$variables"
-
   name=$(graphql_output '.data.calaOutboxImportJobCreate.job.name')
-  [[ "$name" == "rust-example" ]] || exit 1;
+  error_msg=$(graphql_output '.errors[0].message')
+  [[ "$name" == "rust-example" || "$error_msg" =~ duplicate.*jobs_name_key ]] || exit 1;
 
-  background cargo run --bin cala-ledger-example-rust > .rust-example-logs
+  background cargo run --bin cala-ledger-example-rust > .rust-example-logs 2>&1
 
-  sleep 5
+  job_count=$(cat .e2e-logs | grep 'Executing CalaOutboxImportJob importing' | wc -l)
```

Comment:
hmm this is no good - probably should rather have a delete or disable job so that there is always just 1 running... but okay for now I guess.

Reply:
This is probably poorly named. Should be `job_executions_count`. I added it as a way to not arbitrarily, but to instead wait on a job execution before checking for the sync.